doc/source/dev/internals.code-explanations.rst

```diff
@@ -599,10 +596,7 @@ the current output pointer. Thus, the operation performed is
 .. math::
    :nowrap:
 
-   \begin{align*}
-   o[0] & = & i[0] \\
-   o[k] & = & i[k]\textrm{<op>}o[k-1]\quad k=1\ldots N.
-   \end{align*}
+   $$o_k = \begin{cases} & i[0] & k=0 \\\ & i[k] \text{ <op> }o_{k-1} & k=1,\ldots, N \text{.} \end{cases}$$
```

Comment:
This now mixes subscript indexing and square-bracket indexing. Our documentation uses square brackets when describing ndarray indexing, so please revert that change. I am not sure the `cases` notation is appropriate here, since we are iterating over `i` and `o`, the first iteration is special.